pkg/serviceaccount/tokens_controller_test.go

```diff
@@ -372,7 +372,7 @@ func TestTokenCreation(t *testing.T) {
 	for k, tc := range testcases {
 
 		// Re-seed to reset name generation
-		rand.Seed(1)
+		utilrand.Seed(1)
```

Comment:
I really dislike that some random test is dependent on the seed of some distant rng library.  Shouldn't this test be written to be independent of the random portion?